<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>917</width>
    <height>858</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Speech Control by Igor Pro</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButton_start">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>111</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial Black</family>
      <pointsize>11</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 1px solid gray;
border-radius: 10px;
background-color: rgb(217, 226, 255);</string>
    </property>
    <property name="text">
     <string>Старт</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="plainTextEdit">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>160</y>
      <width>400</width>
      <height>215</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 1px solid gray;
border-radius: 10px;
background-color: rgb(239, 247, 255);</string>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>110</y>
      <width>211</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial Black</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Прослушивание</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>110</y>
      <width>211</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial Black</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Действия</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="plainTextEdit_actions">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>160</y>
      <width>400</width>
      <height>215</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 1px solid gray;
border-radius: 10px;
background-color: rgb(239, 247, 255);</string>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_load">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>400</y>
      <width>191</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Список инструментов</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>470</y>
      <width>881</width>
      <height>341</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 1px solid gray;
border-radius: 6px;
background-color: rgb(241, 255, 255);</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_start_tg">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>40</y>
      <width>171</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial Black</family>
      <pointsize>11</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 1px solid gray;
border-radius: 10px;
background-color: rgb(231, 172, 255);</string>
    </property>
    <property name="text">
     <string>Телеграм бот</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_log_clear">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>380</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Очистить лог</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_actions_clear">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>380</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Очистить действия</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_speech">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>130</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Озвучивание</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>60</y>
      <width>541</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:400; color:#ff0000;&quot;&gt;Добавить&lt;/span&gt;&lt;span style=&quot; font-weight:400; font-style:italic; color:#ff0000;&quot;&gt; &amp;lt;съемник&amp;gt; &lt;/span&gt;&lt;span style=&quot; font-weight:400; color:#ff0000;&quot;&gt;МЕСТО&lt;/span&gt;&lt;span style=&quot; font-weight:400; font-style:italic; color:#ff0000;&quot;&gt; &amp;lt;полка 2&amp;gt; &lt;/span&gt;&lt;span style=&quot; font-weight:400; color:#008000;&quot;&gt;КОЛИЧЕСТВО &amp;lt;3&amp;gt;&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>80</y>
      <width>521</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:400; color:#ff0000;&quot;&gt;Убрать&lt;/span&gt;&lt;span style=&quot; font-weight:400; font-style:italic; color:#ff0000;&quot;&gt; &amp;lt;съемник&amp;gt; &lt;/span&gt;&lt;span style=&quot; font-weight:400; color:#ff0000;&quot;&gt;МЕСТО&lt;/span&gt;&lt;span style=&quot; font-weight:400; font-style:italic; color:#ff0000;&quot;&gt; &amp;lt;полка 2&amp;gt; &lt;/span&gt;&lt;span style=&quot; font-weight:400; color:#008000;&quot;&gt;КОЛИЧЕСТВО &amp;lt;2&amp;gt;&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>0</y>
      <width>721</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:11pt; font-weight:600; font-style:italic; color:#ff0000;&quot;&gt;&amp;lt;Действие&amp;gt; &amp;lt;наименование&amp;gt; &lt;/span&gt;&lt;span style=&quot; font-size:11pt; font-weight:600; color:#ff0000;&quot;&gt;МЕСТО&lt;/span&gt;&lt;span style=&quot; font-size:11pt; font-weight:600; font-style:italic; color:#ff0000;&quot;&gt; &amp;lt;название места&amp;gt;&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-size:11pt; font-weight:600; color:#008000;&quot;&gt;КОЛИЧЕСТВО &amp;lt;количество&amp;gt; &lt;/span&gt;&lt;span style=&quot; font-size:11pt; font-weight:600; color:#ff0000;&quot;&gt;ИМЯ &amp;lt;имя&amp;gt; &lt;/span&gt;&lt;span style=&quot; font-size:11pt; font-weight:600; color:#000000;&quot;&gt;(для аренды)&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_rent">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>400</y>
      <width>161</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 1px solid gray;
border-radius: 10px;
background-color: rgb(179, 255, 234);</string>
    </property>
    <property name="text">
     <string>Аренда</string>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
    <property name="default">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>100</y>
      <width>491</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:400; color:#ff0000;&quot;&gt;Аренда&lt;/span&gt;&lt;span style=&quot; font-weight:400; font-style:italic; color:#ff0000;&quot;&gt; &amp;lt;ключ 10&amp;gt; &lt;/span&gt;&lt;span style=&quot; font-weight:400; color:#008000;&quot;&gt;КОЛИЧЕСТВО &amp;lt;2&amp;gt; &lt;/span&gt;&lt;span style=&quot; font-weight:400; color:#ff0000;&quot;&gt;ИМЯ &amp;lt;Василий&amp;gt;&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_search">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>430</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Поиск по базе</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_active_bot">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>10</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt; font-style:italic; color:#804452;&quot;&gt;Бот неактивен!&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_active">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt; font-style:italic; color:#804452;&quot;&gt;Ожидание&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_load_debtors">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>400</y>
      <width>161</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Заёмщики</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>917</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Меню</string>
    </property>
    <widget class="QMenu" name="menu_2">
     <property name="title">
      <string>Настройки</string>
     </property>
     <addaction name="actionSettingsGeneral"/>
     <addaction name="actionSettingsTg"/>
    </widget>
    <addaction name="menu_2"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>Использование</string>
    </property>
    <addaction name="actionInstruction"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_3"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action_2">
   <property name="text">
    <string>фыв</string>
   </property>
  </action>
  <action name="actionSettingsGeneral">
   <property name="text">
    <string>Общие</string>
   </property>
  </action>
  <action name="actionSettingsTg">
   <property name="text">
    <string>Телеграм бот</string>
   </property>
  </action>
  <action name="actionInstruction">
   <property name="text">
    <string>Инструкция</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
